client/clientservice/src/request_service.cpp

```diff
@@ -65,8 +65,13 @@ void RequestServiceCallData::populateResult(grpc::Status status) {
 }
 
 void RequestServiceCallData::sendToConcordClient() {
-  bft::client::Msg msg(request_.request().begin(), request_.request().end());
-
+  bft::client::Msg msg;
+  if (request_.has_typed_request()) {
+    std::string str = request_.typed_request().SerializeAsString();
```

Comment:
As we don't have the replica side implemented here, I'd rather suggest forwarding the embedded message. I.e., we can assume for now that the embedded type is always going to be a `ConcordRequest` or `DamlRequest`.
```suggestion
    std::string str = request_.typed_request().value();
```